closed/src/java.base/share/classes/com/sun/crypto/provider/NativeGaloisCounterMode.java

```diff
@@ -362,6 +377,46 @@ int encryptFinal(byte[] in, int inOfs, int len, byte[] out, int outOfs)
         return (len + tagLenBytes);
     }
 
+    /**
+     * Performs encryption operation for the last time.
+     *
+     * @param src the input buffer with the data to be encrypted
+     * @param dst the offset in <code>in</code>
+     * @return the number of bytes placed into the <code>out</code> buffer
```

Comment:
Please fix this comment: the parameter is named `dst`.

Reply:
I edited comments in both: `encryptFinal` and `decryptFinal` to match the actual parameters.